src/System.IO/tests/Stream/Stream.NullTests.cs

```diff
@@ -5,31 +5,91 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
+using System.Linq;
+using System.Threading;
 using System.Threading.Tasks;
 using Xunit;
 
 namespace System.IO.Tests
 {
-    public class NullCtorTests
+    public class NullTests
     {
+        [Theory]
+        [InlineData(20)]
```

Comment:
Did that so the tests could verify that calling `Dispose` wouldn't have any side effects; e.g. calling `Dispose` twice wouldn't throw on the second time. In hindsight maybe making it a `Theory` and adding a parameter for the number of iterations was a bit overkill; I'll simply copy-paste the method call and add a note explaining why it's being repeated.